manifests/master/pdf_stack.pp

```diff
@@ -1,7 +1,7 @@
 # Dependencies for PDF rendering
 class classroom::master::pdf_stack {
 
-  $enabled = $classroom::offline ? {
+  $enabled = defined('$classroom::offline') and $classroom::offline ? {
```

Comment:
I haven't tested it, but to get the current code testable, you *should* be able to do something like

```ruby
let(:pre_condition) { 'class classroom { $offline = true }; include classroom' }
```

The better fix is refactoring, but that doesn't have to happen today and this doesn't add anything to our tech debt.